tsl/test/expected/dist_compression.out

```diff
@@ -324,11 +324,20 @@ ORDER BY chunk_name, node_name;
  _timescaledb_internal | _dist_hyper_1_3_chunk |        8192 |       32768 |           0 |       40960 | data_node_3
 (6 rows)
 
-SELECT * FROM hypertable_detailed_size('compressed'::regclass) 
+SELECT * FROM hypertable_detailed_size('compressed'::regclass);
  table_bytes | index_bytes | toast_bytes | total_bytes |  node_name  
 -------------+-------------+-------------+-------------+-------------
        16384 |       65536 |           0 |       81920 | data_node_1
        16384 |       65536 |           0 |       81920 | data_node_3
        16384 |       65536 |           0 |       81920 | data_node_2
 (3 rows)
 
+------------------------------------------------------
+-- Test compression policy on a distributed hypertable
+------------------------------------------------------
+INSERT INTO compressed VALUES 
+(now()::TIMESTAMPTZ, 1, 0.1), (now()::TIMESTAMPTZ, 2, 0.2);
+\set ON_ERROR_STOP 0
+SELECT add_compression_policy('compressed', '60d'::interval);
```

Comment:
compression is enabled. But this fails. I think it is better to add support for the compression policy in the same PR. Or not add the test here.

Reply:
@gayyappan How can this error happen if the distributed hypertable is marked with the compression enabled? I suspect that the alter statement, which enables compression, is just propagated to data nodes, but is not set on the distributed hypertable. What am I missing?